modules/bitgo/test/v2/unit/internal/opengpgUtils.ts

```diff
@@ -32,6 +37,26 @@ describe('OpenGPG Utils Tests', function () {
           email: 'other@username.com',
         },
       ],
+      curve: 'secp256k1',
+    });
+  });
+
+  describe('createShareProof', function () {
+    it('should create a share proof', async function () {
+      const uValue = '36cba3316f8e8918c2ab7284c2a9b541efd7705e1121f6f5a144de1ca8543708';
```

Comment:
you could make this random, or get it from a real keyshare

Reply:
good call. i copy pasted this from a real key share, but might as well randomize.